app/Enums/UserRole.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace App\Enums;
+
+enum UserRole: string {
+    case Administrador = "Administrador";
```

Comment:
Yo pondría el case en mayúsculas y en ingles por seguir la convención del resto de las enumeraciones, al final es como una variable constante.
```suggestion
    case ADMIN = "Administrador";
```